.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+
+# Contributing to elrond-go
+
+If you are unfamiliar with the workflow of contributing to github, you can refer to this [this article](https://github.com/firstcontributions/first-contributions/blob/master/README.md)
+
+## Fork & clone this repository
+
+The development should happen in a personal fork, cloned on the local machine.
+
+## Use development branch
+
+External contributions should happen against the `development` branch.
+
+Other branches may be used by the team members in accordance to internal decisions.
+
+## Use linter
+
+Make sure the code is well-formatted and aligned before opening a PR. Some linters such `gofmt` can be used, or code style
+can be proactively checked while written code when using an IDE.
+
+## Writing tests
+
+In case of adding a new feature, make sure you follow these guidelines:
+- Attempt to write some unit tests which verify the component in isolation and mock the external components around it
+- If the component relies on many other external components which are difficult to mock, write integrations tests for it
+- Try to cover both expected functionality but also verify that the code fails in expected ways
```

Comment:
Remove **try to**: Cover both expected functionality...

Reply:
done